steve/tests/test_steve_cmd.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+
+from __future__ import print_function
+
+import os
+import subprocess
+
+import pytest
+
+class TestSteveCmd:
+    def test_create_project(self, tmpdir, monkeypatch):
+        # this changes to the temporary pytest directory and back after
+        # the test has been run
+        monkeypatch.chdir(tmpdir)
+        proj = 'testprj'
+        res = subprocess.check_output(['steve-cmd', 'createproject', proj])
+        # you should be able to find the project from the last run with
+        # find /tmp/pytest-current/
+        # note the final "/" as this is a link to
+        #   /tmp/pytest-NNN/test_create_project0
+        assert os.path.exists(os.path.join(proj, 'steve.ini'))
```

Comment:
For now it can be used to actually test if the implementation of click works. At the moment the click PR/branch fails to perform even this basic task. Once click integration actually does something its facilities can be used and this test can of course be removed/replaced.